monri/src/main/AndroidManifest.xml

```diff
@@ -2,8 +2,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.monri.android">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+
     <application>
-        <activity android:name=".activity.ConfirmPaymentActivity" />
+        <activity android:name=".activity.ConfirmPaymentActivity"
+            android:theme="@style/Theme.AppCompat.Light"/>
```

Comment:
@adnanomerovic why's this required?

Reply:
Part - android:theme="@style/Theme.AppCompat.Light"

Reply:
I got exception when I'm trying to launch activity in androidTest (ActivityScenario.launch() throw exception)